modules/extensions/galasa-extensions-parent/dev.galasa.cps.etcd/src/main/java/dev/galasa/cps/etcd/internal/Etcd3ConfigurationPropertyStore.java

```diff
@@ -81,7 +82,8 @@ public Etcd3ConfigurationPropertyStore(URI cpsUri) {
         HashMap<String, String> returnValues = new HashMap<>();
         
         ByteSequence bsKey = ByteSequence.from(prefix, UTF_8);
-        GetOption option = GetOption.newBuilder().withPrefix(bsKey).build();
+        ByteSequence prefixEnd = OptionsUtil.prefixEndOf(bsKey);
```

Comment:
To avoid a deprecated method.

Reply:
Looks like `withPrefix` was deprecated in favour of `isPrefix()` (see https://github.com/etcd-io/jetcd/blob/main/jetcd-core%2Fsrc%2Fmain%2Fjava%2Fio%2Fetcd%2Fjetcd%2Foptions%2FGetOption.java#L308) - does `prefixEndOf` do the same thing?